src/com/epam/task/module4/theSimplestClassesAndObjects/task1/service/Service.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.task.module4.theSimplestClassesAndObjects.task1.service;
+
+import static java.lang.Math.max;
+
+public class Service {
+    public static Integer getSumNumbers(Integer first, Integer second) {
```

Comment:
Interesting, why you have decided to move the logic outside of Test1 class in this task.